doc/test-framework/writing-e2e-tests.md

```diff
@@ -194,31 +194,71 @@ functions will automatically be run since they were deferred when the TestCtx wa
 
 ```shell
-$ operator-sdk test --test-location ./test/e2e --go-test-flags "-v -parallel=2"
+$ operator-sdk test local ./test/e2e --go-test-flags "-v -parallel=2"
 ```
 
-For more documentation on the `operator-sdk test` command, see the [SDK CLI Reference][sdk-cli-ref] doc.
+For more documentation on the `operator-sdk test local` command, see the [SDK CLI Reference][sdk-cli-ref] doc.
 
 For advanced use cases, it is possible to run the tests via `go test` directly. As long as all flags defined
 in [MainEntry][main-entry-link] are declared, the tests will run correctly. Running the tests directly with missing flags
-will result in undefined behavior. This is an example `go test` equivalent to the `operator-sdk test` example above:
+will result in undefined behavior. This is an example `go test` equivalent to the `operator-sdk test local` example above:
 
 ```shell
-# Combine rbac and operator manifest into namespaced manifest
-$ cp deploy/rbac.yaml deploy/namespace-init.yaml
+# Combine sa, rbac, operator manifest into namespaced manifest
+$ cp deploy/sa.yaml deploy/namespace-init.yaml
+$ echo -e "\n---\n" >> deploy/namespace-init.yaml
+$ cat deploy/rbac.yaml >> deploy/namespace-init.yaml
 $ echo -e "\n---\n" >> deploy/namespace-init.yaml
 $ cat deploy/operator.yaml >> deploy/namespace-init.yaml
 # Run tests
 $ go test ./test/e2e/... -root=$(pwd) -kubeconfig=$HOME/.kube/config -globalMan deploy/crd.yaml -namespacedMan deploy/namespace-init.yaml -v -parallel=2
 ```
 
+### Cluster
+
+Another way to run the tests is from within a kubernetes cluster. To do this, you first need to build an image with
+the testing binary embedded by using the `operator-sdk build` command and using the `--enable-tests` flag to enable tests:
+
+```shell
+$ operator-sdk build quay.io/example/memcached-operator:v0.0.1 --enable-tests
+```
+
+Note that the namespaced yaml must be up to date before running this command. The `build` subcommand will warn you
+if it finds a deployment in the namespaced manifest with an image that doesn't match the argument you provided. The
+`operator-sdk build` command has other flags for configuring the tests that can be viewed with the `--help` flag
+or at the [SDK CLI Reference][sdk-cli-ref].
+
+Once the image is ready, the tests are ready to be run. To run the tests, make sure you have all global resources
+and a namespace with proper rbac configured:
+
+```shell
+$ kubectl create -f deploy/crd.yaml
+$ kubectl create namespace memcached-test
+$ kubectl create -f deploy/sa.yaml -n memcached-test
+$ kubectl create -f deploy/rbac.yaml -n memcached-test
+```
+
+Once you have your environment properly configured, you can start the tests using the `operator-sdk test cluster` command:
+
+```shell
+$ operator-sdk test cluster quay.io/example/memcached-operator:v0.0.1 --namespace memcached-test
+
+Example Output:
+Test Successfully Completed
+```
+
+If the tests fail, the command will output the errors in the standard `go test` loggin/error format.
```

Comment:
How about a little clarification on how the command actually runs the test. So its clear how this is different from `test local`. e.g:

The test command will deploy a test pod in the given namespace that will run the e2e tests packaged in the image.
The command will wait until the tests succeed(pod phase=`Succeeded `) or fail(pod phase=`Failed `).
If the tests fail, the command will output the test pod logs which should be the standard `go test` error logs.